.github/workflows/release.yml

```diff
@@ -114,12 +114,12 @@ jobs:
           *These runtimes were built with **$(SRTOOL rustc)** using **[$(SRTOOL gen)](https://github.com/paritytech/srtool)***
 
           To replicate the build, use the following command:
-          ```sh
-          srtool build \
-                 --root --profile production \
-                 --package CRATE_NAME --runtime-dir PATH_TO_CRATE \
+          \`\`\`sh
+          srtool build
```

Comment:
The last time i tried i had to also explicitly define the Rustc version like `export RUSTC_VERSION=1.74.0; srtool build ...`.

Reply:
This env variable does not seem to exist in the cli tool: https://github.com/search?q=repo%3Achevdor%2Fsrtool-cli+RUSTC_VERSION&type=code

Reply:
Hm, maybe the env is passed through... going to check again next release.